### DISCUSSION TITLE
Query string handling parses string value as an exponent

### DISCUSSION THREAD

**Discussion by V13Axel** [livewire/livewire#7156] [7. Bugs (archived)]
### Livewire version

v3.0.10

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/V13Axel/livewire-query-string-exponent-bug

### Steps To Reproduce

In the demo repository, just clone it, run `composer install` and then visit `/`.

Otherwise:
1. Create a Livewire component, give it a route. Let's say `/your/component/route`.
2. Give it the following:
```php
#[Url]
public string $search = '';

public function mount() {
    dd($this->search, request('search'));
}
```
3. Visit /your/component/route?search=123e5
4. Observe the difference between the component property value `12300000`, and the request parameter value `123e5`.

### Additional information

This was encountered in a search field, where the value is expected to be, and remain, exactly what the user entered - In this instance, a "case number" of sorts that can be alphanumeric. 

The user's search was `104E3058`, and since a 3,058 digit exponent is much too large for PHP, it results in any references to `$this->search` (such as passing to an Eloquent query) receiving the value "INF" rather than the search string the user entered.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by kermitsxb** [comment]
I'm having the same issue, does this bug already have an issue ?
